web/src/pages/index.tsx

```diff
@@ -147,95 +149,100 @@ const Home = (props: Props): ReactElement => {
   };
 
   return (
-    <PageSkeleton landingPage={true}>
-      <NavBar landingPage={true} />
-      <main data-testid="main">
-        <Hero isWelcomePage={props.isWelcomePage} />
-        <section ref={sectionHowItWorks} aria-label="How it works">
-          <div className="minh-mobile margin-top-2 desktop:margin-top-neg-205  padding-bottom-6 text-center bg-base-extra-light">
-            <h2 className="text-accent-cool-darker h1-styling margin-bottom-6 padding-top-6">
-              {landingPageConfig.section4HeaderText}
-            </h2>
-            <div
-              className={`flex ${
-                isDesktopAndUp ? "flex-row flex-justify-center" : "flex-column flex-align-center"
-              } text-accent-cool-darker`}
-            >
-              <div className="margin-x-3">
-                <img className="" src="/img/Landing-documents.svg" alt="" role="presentation" />
-                <div className="text-accent-cool-darker width-card margin-top-2">
-                  {landingPageConfig.section4FirstIconText}
-                </div>
-              </div>
+    <>
+      <NextSeo title={`${seoTitle.title} - ${Config.seoAndAccessibilityPageTitles.homePageTitle}`} />
+      <PageSkeleton landingPage={true}>
+        <NavBar landingPage={true} />
+        <main data-testid="main">
+          <Hero isWelcomePage={props.isWelcomePage} />
+          <section ref={sectionHowItWorks} aria-label="How it works">
```

Comment:
I know this was pre-existing, but I would recommend moving this label to config as well.

Reply:
Where in config were you imagining? I can look around for a good spot but I'm wondering if you had a place in mind

Reply:
I did not have a specific place in mind; I trust wherever you place it.